system/netdata-openrc.in

```diff
@@ -38,13 +38,13 @@ depend() {
 }
 
 start_post() {
-	if [[ ! -f @sysconfdir_POST@/netdata/netdata.conf ]]; then
+	if [ ! -f @sysconfdir_POST@/netdata/netdata.conf ]; then
 		ebegin "Downloading default configuration to @sysconfdir_POST@/netdata/netdata.conf"
 		sleep 2
 		curl -s -o @sysconfdir_POST@/netdata/netdata.conf.new "${NETDATA_CONFIG_URL}"
 		ret=$?
-		if [[ $ret -eq 0 && -s @sysconfdir_POST@/netdata/netdata.conf.new ]]; then
-			mv @sysconfdir_POST@/netdata/netdata.conf{.new,}
+		if [ $ret -eq 0 && -s @sysconfdir_POST@/netdata/netdata.conf.new ]; then
+			mv @sysconfdir_POST@/netdata/netdata.conf @sysconfdir_POST@/netdata/netdata.conf.new
```

Comment:
I think this should be the other way around: move the `.new` file to the plain one.

Reply:
Thanks! I've pushed a fix.